common/men_at_arms_types/00_maa_types.txt

```diff
@@ -169,9 +203,68 @@ onager = {
 	
 	stack = 10
 	allowed_in_hired_troops = no
+	icon = siege_weapon
+}
+###WARCRAFT###
+giant_unit = {
+	type = massive_infantry
+	
+	damage = 36
+	toughness = 35
+	pursuit = 0
+	screen = 13
+	
+	terrain_bonus = {
+		wetlands = { damage = -15}
+		jungle = { damage = -10 }
+		mountains = { damage = -7}
+
+	}
+
+	counters = {
+		skirmishers = 2
+	}
+
+	buy_cost = { gold = massive_infantry_recruitment_cost }
+	low_maintenance_cost = { gold = massive_infantry_low_maint_cost }
+	high_maintenance_cost = { gold = massive_infantry_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = culture_ai_weight_heavy_cavalry }
+	icon = massive_infantry
+
 }
 
+
 #Medieval Era
+###WARCRAFT###
+aerial_unit = {
+	type = light_aerial
+	
+	damage = 20
+	toughness = 12
+	pursuit = 30
+	screen = 5
+	
+	terrain_bonus = {
+		forest = { damage = -10 pursuit = -10 }
+		jungle = { damage = -10 pursuit = -10 }
+
+	}
+
+	counters = {
+		massive_aerial = 2
```

Comment:
Why you changed it? It breaks the balance. `light_aerial` is countered by one unit with a factor of 1.

Reply:
Some units only counter 1 other by a factor of one, while most other units counter two or three

Reply:
Is that ok?

Reply:
> Is that ok?

`light_aerial` is balanced on the docs and doesn't need a buff. That's what I meant.

Reply:
> Some units only counter 1 other by a factor of one, while most other units counter two or three

It counters and countered by one unit. Thus it's balanced. If you change the factor to 2, it counters more effectively.